cli/params.yaml

```diff
@@ -0,0 +1,34 @@
+params:
+  - name: s3-bucket-name
+    value: rhoai-abdalla-pipelines
+  - name: git-containerfile-repo
+    value: https://github.com/opendatahub-io/ai-edge.git
+  - name: git-containerfile-revision
+    value: "main"
+  - name: containerfileRelativePath
+    value: pipelines/containerfiles/Containerfile.seldonio.mlserver.mlflow
```

Comment:
As one example, I'm not sure if this would fit as _model_ metadata, since a hypothetical user may want to package the same model into two different images with different model servers (think of deployment to two different edge locations, with slightly different hardware, or something), which would mean two different images would be created from two different Containerfiles.

I don't have a great handle on the hierarchy of things in the model registry yet though, and where this fits in; so what I'm saying might not make 100% sense.

Reply:
> As one example, I'm not sure if this would fit as _model_ metadata, 

As I mentioned above, I think we should aim to have a properly designed API that contains metadata with proper semantic meanings. So, this definitely needs to go back to the whiteboard, as currently, it's just a bunch of arbitrary key-value pairs.

> since a hypothetical user may want to package the same model into two different images with different model servers (think of deployment to two different edge locations, with slightly different hardware, or something), which would mean two different images would be created from two different Containerfiles.

I see your point. However, if we're talking about container images that combine a model server and the model, then I believe each variant should be considered a different artifact for the model. In such cases, some of the metadata will be associated only with a specific kind of artifact, and others will stay at the version level.